src/Raven.Studio/typescript/common/shell/accessManager.ts

```diff
@@ -113,19 +113,27 @@ class accessManager {
             case "Operator":
                 return this.isOperatorOrAbove();
         }
-        
+
         const activeDatabaseAccess = this.activeDatabaseAccessLevel();
 
         if (!activeDatabaseAccess) {
             return false;
         }
 
-        if (activeDatabaseAccess === 'Read' &&
-            (requiredAccess === "DatabaseReadWrite" || requiredAccess === "DatabaseAdmin")) {
+        return this.canHandleOperationDatabaseLevelCheck(activeDatabaseAccess, requiredAccess as databaseAccessLevel);
```

Comment:
can we avoid op operator?

Reply:
removed the 'as' operator

Reply:
great - we should avoid as + control flow in typescript should automatically infer based on code flow that it has to be databaseAccessLevel. That's the beauty of typescript.